src/c4v/scraper/persistency_manager/json_storage_manager.py

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
Is this our local parallel implementation to the BigQuery one?

if that is the case, then this is not a good option.
Can we change this by an implementation based on SQLite please: https://docs.python.org/3/library/sqlite3.html

Reply:
This is for our local parallel implementation, to do local testing, but I'll agree that it's inefficient, I'll switch it to an SQLite based one.